apps/avatax/src/pages/api/webhooks/order-cancelled.ts

```diff
@@ -33,7 +33,8 @@ export default wrapWithLoggerContext(
       if (!payload.order) {
         const error = new Error("Insufficient order data");
 
-        Sentry.captureException(error);
```

Comment:
Im not sure if Sentry shouldnt be there? I guess ORDER_CANCELLED should always have order data. If it doesn't, it means something broke

Reply:
Order should be non-nullable. Issue tracked under https://linear.app/saleor/issue/SHOPX-395. I think we can keep this removed and after fix is delivered remove this `if` entirely.